Lib/pkgutil.py

```diff
@@ -252,7 +252,8 @@ def load_module(self, fullname):
         return mod
 
     def get_data(self, pathname):
-        return open(pathname, "rb").read()
+        with open(pathname, "rb") as file:
+            return file.read()
```

Comment:
Top of the module says:

```
# NOTE: This module must remain compatible with Python 2.3, as it is shared
# by setuptools for distribution with Python 2.3 and up.
```

Would you mind investigating if that’s still true?

Reply:
I care way more about leaking resources than about compatibility with Python < 2.5. I believe that this isn't something we should be cautious about.

Reply:
I don’t disagree!  Maybe remove the comment in that case, so that future reviews don’t ask the same questions.

Reply:
I checked setuptools and it doesn’t include pkgutil in the current version.  I removed the comment and would be happy to merge.